oauth2-server/src/test/java/com/clouway/oauth2/SerializeBearerTokensTest.java

```diff
@@ -17,26 +17,39 @@
 
   @Test
   public void happyPath() throws IOException {
+    String encodedIdToken="eyJpc3MiOiJwcm92aWRlci5zb21lZG9tYWluLmNvbSIsDQogImF0X2hhc2giOiJISzZFX1A2RGg4WTkzbVJOdHNEQj" +
```

Comment:
You can use ::id token:: as value as it's value is not relevant with the test. You are just verifying that there is a property id_token and it's value is added to the response.